compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaFunctionSymbol.java

```diff
@@ -134,7 +134,8 @@ public FunctionSymbolBuilder withAnnotation(AnnotationSymbol annot) {
         @Override
         public BallerinaFunctionSymbol build() {
             return new BallerinaFunctionSymbol(this.name, this.qualifiers, this.annots,
-                                               this.typeDescriptor, (BInvokableSymbol) this.bSymbol, this.context);
+                    this.typeDescriptor, (BInvokableSymbol) this.bSymbol,
+                    this.context);
```

Comment:
This as well.

Reply:
Btw, this seems to be just a formatting change?

Reply:
I'll correct it. This happened due to two commits including a revert commit